File: 백준/Bronze/10810. 공 넣기/공 넣기.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int N = sc.nextInt();
        int M = sc.nextInt();
        
        int Box[] = new int[N];
        
        for (int a = 0; a < M; a++) {
            int i = sc.nextInt();
            int j = sc.nextInt();
            int k = sc.nextInt();
            
            for (int b = i - 1; b < j; b++) {
                Box[b] = k;
            }
        }
        
        for (int c = 0; c < Box.length; c++) {
            System.out.print(Box[c] + " ");
        }
        
        sc.close();
    }
}

File: 백준/Bronze/10810. 공 넣기/공 넣기.c
#include <stdio.h>

int main() {
    int N, M;
    scanf("%d %d", &N, &M);
    
    int Box[N];
    for (int i = 0; i < N; i++) {
        Box[i] = 0;
    }
    
    for (int a = 0; a < M; a++) {
        int i, j, k;
        scanf("%d %d %d", &i, &j, &k);
        
        for (int b = i - 1; b < j; b++) {
            Box[b] = k;
        }
    }
    
    for (int c = 0; c < N; c++) {
        printf("%d ", Box[c]);
    }
    
    return 0;
}